CASMNET-1745 BICAN mgmt switch config (#2212)
Cray-HPE/docs-csm: operations/network/management_network/bican_enable.md

Setting SystemDefaultRoute to CHN
```

### Configure Management Network

> If the system is moving from the CAN to CHN the management network will need configuration changes to allow connectivity.

Reviewer: ```suggestion
If the system is moving from the CAN to CHN, then the management network will need configuration changes to allow connectivity.
```

> 1. Configure management network switches and customer edge switches. [BICAN switch configuration](bican_switch_configuration.md)

Reviewer: ```suggestion
Refer to [BICAN switch configuration](bican_switch_configuration.md) to configure management network switches and customer edge switches.
```

> Concretely, users on a running UAN may be transitioned from the CMN to the new CAN between the two following upgrade points:

Reviewer: ```suggestion
Concretely, users on a running UAN may be transitioned from the CMN to the new CAN before [UANs are booted with new images](../../boot_orchestration/Boot_UANs.md).
```

> 1. Before [UANs are booted with new images](../../boot_orchestration/Boot_UANs.md).

Reviewer: ```suggestion
```
